scripts/install-windows-native.ps1

```
@@ -0,0 +1,515 @@
+#>
+
+[CmdletBinding()]
+param(
+    [ValidateSet('Install', 'Repair', 'Uninstall')]
+    [string]$Action = 'Install',
+
+    [string]$ManifestPath,
+
+    [string]$PayloadRoot,
+
+    [string]$InstallRoot = (Join-Path
```

PSScriptAnalyzer (1.25.0)

[error] Missing closing ')' in expression.

(MissingEndParenthesisInExpression)